.github/workflows/build-ros2-ws-action.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [master]  
+jobs:
+  build-and-test:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        run: git clone -b feature/fix-using-grpc https://github.com/ichiro-its/aruku.git
+
+      - name: Checkout Kansei
+        run: git clone https://github.com/ichiro-its/kansei.git                 
+
+      - name: Checkout Kansei Interfaces
+        run: git clone https://github.com/ichiro-its/kansei_interfaces.git         
+      
+      - name: Checkout Aruku Interfaces
+        run: git clone https://github.com/ichiro-its/aruku_interfaces.git     
+
+      - name: Checkout Keisan
+        run: git clone https://github.com/ichiro-its/keisan.git        
+
+      - name: Checkout tachimawari
+        run: git clone https://github.com/ichiro-its/tachimawari.git  
+
+      - name: Checkout tachimawari interfaces
+        run: git clone https://github.com/ichiro-its/tachimawari_interfaces.git    
+
+      - name: Setup cmake        
+        run: sudo apt install -y build-essential autoconf libtool pkg-config cmake
```

Comment:
Can you verify if all these packages are required in this context?

It seems that most packages here are already installed in the latest Ubuntu runner (see [this](https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md)).

Maybe we can remove some packages from this list?

Reply:
I try to remove this from this step, mas. If I encounter an error, I will put it back.